iina/MenuController.swift

```diff
@@ -8,36 +8,65 @@
 
 import Cocoa
 
-fileprivate func sameKeyAction(_ lhs: [String], _ rhs: [String], _ normalizeLastNum: Bool, _ numRange: ClosedRange<Double>?) -> (Bool, Double?) {
+fileprivate func sameKeyAction(_ lhs: [String], _ rhs: [String], _ normalizeLastNum: Bool, _ numRange: ClosedRange<Double>?) -> (Bool, Double?, Any?) {
   var lhs = lhs
-  if lhs.first == "seek" && (lhs.last == "exact" || lhs.last == "keyframe") {
-    lhs = [String](lhs.dropLast())
+  var extraData: Any? = nil
+  if lhs.first == "seek" {
+    if let last = lhs.last {
+      if let _ = Double(last) {
+        // "relative+keyframes" is implicit default.
+        extraData = Preference.SeekOption.defaultValue
+      } else {
+        // matching values are one of: "relative", "keyframes", "exact", "relative+keyframes", "relative+exact"
+        let splitArray = last.split(whereSeparator: { $0 == "+" })
+        if splitArray.count == 2 && splitArray[0] == "relative" {
```

Comment:
Here you assumed "relative" always appears first, however it seems things like "keyframes+relative" are also valid mpv flags. I tried to add `seek -5 keyframes+relative` and `seek 5 relative+keyframes`, both shortcuts can be used in IINA, but only the later one is recognized as the same action as in the default menu.